### ISSUE TITLE
Colour contrast too low on workspace buttons

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1193]
The white text on orange background we use for buttons in the workspace, management page and login pages fails colour contrast checks. Not sure what colour we should change this to though as the orange fits with the Xerte branding. Oranges that do pass with white text are quite brown in colour.

**Comment by FayCross** [comment]
@torinfo I have changed size and weight of text on these buttons so the contrast is ok (WCAG AA). I've checked most places that buttons with these classes occur but they are also used on dashboard. You might want to check that buttons with class xerte_button_c_no_width look ok on dashboard as they are now larger so may mess up the layout (I don't know how i would check this in xampp). Thanks
